src/app/space/wizard/space-wizard.component.ts

```diff
@@ -120,16 +128,26 @@ export class SpaceWizardComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
-    const srumTemplates = this.spaceTemplates.filter(template => template.name === 'Scenario Driven Planning');
-    if (srumTemplates && srumTemplates.length > 0) {
-      this.selectedTemplate = srumTemplates[0];
-    }
     this.context.current.subscribe((ctx: Context) => {
       if (ctx.space) {
         this.currentSpace = ctx.space;
         console.log(`ForgeWizardComponent::The current space has been updated to ${this.currentSpace.attributes.name}`);
       }
     });
+    this.spcaeTemplateService.getSpaceTemplates()
+      .subscribe((templates: ProcessTemplate[]) => {
+        this.spaceTemplates = templates.filter(t => t.attributes['can-construct']);
+        this.selectedTemplate = !!this.spaceTemplates.length ? this.spaceTemplates[0] : null;
+      }, () => {
+        this.spaceTemplates = [{
+          id: '0',
+          attributes: {
+            name: 'Default template',
```

Comment:
What is the default template? Does this text appear when the service did not return anything?

Reply:
Yes, it gets rendered when nothing is received from the backend.